mmv1/third_party/terraform/utils/provider_handwritten_endpoint.go.erb

```diff
@@ -151,40 +136,27 @@ var ServiceUsageCustomEndpointEntry = &schema.Schema{
 	ValidateFunc: validateCustomEndpoint,
 	DefaultFunc: schema.MultiEnvDefaultFunc([]string{
 		"GOOGLE_SERVICE_USAGE_CUSTOM_ENDPOINT",
-	}, ServiceUsageDefaultBasePath),
+	}, DefaultBasePaths[ServiceUsageBasePathKey]),
 }
 
-var StorageTransferDefaultBasePath = "https://storagetransfer.googleapis.com/v1/"
 var StorageTransferCustomEndpointEntryKey = "storage_transfer_custom_endpoint"
 var StorageTransferCustomEndpointEntry = &schema.Schema{
 	Type:         schema.TypeString,
 	Optional:     true,
 	ValidateFunc: validateCustomEndpoint,
 	DefaultFunc: schema.MultiEnvDefaultFunc([]string{
 		"GOOGLE_STORAGE_TRANSFER_CUSTOM_ENDPOINT",
-	}, StorageTransferDefaultBasePath),
+	}, DefaultBasePaths[StorageTransferBasePathKey]),
 }
 
-var BigtableAdminDefaultBasePath = "https://bigtableadmin.googleapis.com/v2/"
 var BigtableAdminCustomEndpointEntryKey = "bigtable_custom_endpoint"
 var BigtableAdminCustomEndpointEntry = &schema.Schema{
 	Type:         schema.TypeString,
 	Optional:     true,
 	ValidateFunc: validateCustomEndpoint,
 	DefaultFunc: schema.MultiEnvDefaultFunc([]string{
 		"GOOGLE_BIGTABLE_CUSTOM_ENDPOINT",
-	}, BigtableAdminDefaultBasePath),
-}
-
-var EventarcDefaultBasePath = "https://eventarc.googleapis.com/v1/"
```

Comment:
Lets not remove this yet, I have a PR to fix some generated base path issues coming

Reply:
left in after discussion